src/diffusers/pipelines/__init__.py

```diff
@@ -448,6 +456,13 @@
             LEditsPPPipelineStableDiffusion,
             LEditsPPPipelineStableDiffusionXL,
         )
+        from .marigold import (
+            MarigoldDepthOutput,
+            MarigoldDepthPipeline,
+            MarigoldImageProcessor,
+            MarigoldNormalsOutput,
+            MarigoldNormalsPipeline,
```

Comment:
```suggestion
            MarigoldDepthPipeline,
            MarigoldNormalsPipeline,
```


This is sufficient!
```python
from diffusers.pipelines.marigold import MarigoldImageProcessor
```

also, if the user has already created a marigold pipeline, they can always get the image processor with `pipe.image_processor` and use all its visualization methods without additional import!

Reply:
Done